test/unit/rules/require-lang-attribute-test.js

```diff
@@ -4,8 +4,35 @@ generateRuleTests({
   name: 'require-lang-attribute',
 
   config: true,
-
-  good: ['<html lang="en"></html>', '<html lang="en-US"></html>', '<html lang={{lang}}></html>'],
+  good: [
+    '<html lang="en"></html>',
+    '<html lang="en-US"></html>',
+    '<html lang={{lang}}></html>',
```

Comment:
We need to make sure we have a test for the default behavior of allowing invalid values when an option isn't specified.
```suggestion
    '<html lang={{lang}}></html>',
    '<html lang="hurrah"></html>'
```